src/components/board/TagsTabSidebard.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+	<div>
+		<ul class="labels">
+			<li v-for="label in labels" :key="label.id">
+				<template v-if="editingLabelId === label.id">
+					<input v-model="editingLabel.title">
+					<compact-picker :value=editingLabel.color @input="updateColor" :palette=defaultColors></compact-picker>
+					<button class="icon-checkmark" @click="updateLabel(label)" :disabled="!editLabelObjValidated" 
+					v-tooltip="{content: missingDataLabel, show: !editLabelObjValidated, trigger: 'manual' }" />
+
+					<button v-tooltip="t('deck', 'Cancel')" class="icon-close" @click="editingLabelId = null" />
+				</template>
+				<template v-else>
+					<span :style="{ backgroundColor: `#${label.color}`, color:textColor(label.color) }" class="label-title">
+						<span v-if="label.title">{{ label.title }}</span><i v-if="!label.title"><br></i>
+					</span>
+					 <button v-tooltip="t('deck', 'Edit')" class="icon-rename" @click="clickEdit(label)" />
+					<button v-tooltip="t('deck', 'Delete')" class="icon-delete" @click="deleteLabel(label.id)" />
+				</template>
+			</li>
+			
+			<li v-if="addLabel">
+				<template>
+				<input v-model="addLabelObj.title">
+				<compact-picker value="#fff" @input="updateColor" :palette=defaultColors></compact-picker>
```

Comment:
White shouldn't be the default color, we should pick the first one of the color picker palette. Also please wrap the defaultColors in quotes.